content/blog/2016/2016-01-05-new-website.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "A new Jenkins website"
+tags:
+- www
+author: rtyler
+---
+
+When link:https://github.com/rtyler[I] first started working on the Jenkins
+website (then called by a different name) I selected
```

Comment:
fwiw, parens are fine, IMO, but it still would need a comma after the parens, since "When I first started working on the Jenkins website I selected" would still really want a comma after "website" anyway. =)